openage/convert/slp.pyx

```diff
@@ -1023,7 +1023,7 @@ cdef numpy.ndarray determine_rgba_matrix(vector[vector[pixel]] &image_matrix,
             else:
                 if px_type == color_player:
                     # mark this pixel as player color
-                    alpha = 254
+                    alpha = 255
```

Comment:
this should remain 254 as we currently filter for 254 in `libopenage/game_renderer.cpp`:
```
glUniform1f(teamcolor_shader::alpha_marker_var, 254.0/255.0)
```

also, this would require people to reconvert the graphics.